_genonce.sh

```diff
@@ -12,6 +12,13 @@ else
 	txoption="-tx n/a"
 fi
 
+# The NLM server is currently running at R3 and thus not compatible with our current efforts
```

Comment:
Could you expand on this comment?

Reply:
> Could you expand on this comment?

I included the two potential considerations for external terminology servers for convenience in case we decide to start using one of them. However, that NLM server only supports STU3. I think the plans are to update it to R4 some time this summer.